Jenkinsfile

```diff
@@ -1,31 +1,129 @@
-@Library('existing-build-control')
+@Library('corda-shared-build-pipeline-steps')
 import static com.r3.build.BuildControl.killAllExistingBuildsForJob
+import groovy.transform.Field
 
 killAllExistingBuildsForJob(env.JOB_NAME, env.BUILD_NUMBER.toInteger())
 
+@Field
+String mavenLocal = 'tmp/mavenlocal'
+
+def nexusDefaultIqStage = "build"
+
+/**
+ * make sure calculated default value of NexusIQ stage is first in the list
+ * thus making it default for the `choice` parameter
+ */
+def nexusIqStageChoices = [nexusDefaultIqStage].plus(
```

Comment:
What's the rationale for `<something>.plus(...).minus(<something>)`?

Reply:
Copy and paste error thanks ...  Removed some redundant lines here. 

No need for Minus and also a duplicate 'build' entry in the list as this is already set via `nexusDefaultIqStage`